src/distr/bernoulli.rs

```diff
@@ -136,6 +136,16 @@ impl Bernoulli {
         let p_int = ((f64::from(numerator) / f64::from(denominator)) * SCALE) as u64;
         Ok(Bernoulli { p_int })
     }
+
+    #[inline]
+    /// Returns the probability (`p`) of the distribution.
+    pub fn p(&self) -> f64 {
+        if self.p_int == ALWAYS_TRUE {
+            1.0
+        } else {
+            f64::from(self.p_int) / SCALE
```

Comment:
Since `p_int` is constructed in `new` or `from_ratio` via `as` cast from `f64`, we can safely use `as` to cast back again (there may be some loss of precision, particularly if a very small value were passed into `new`).